qml/PingSlider.qml

```diff
@@ -49,8 +48,27 @@ RowLayout {
         visible: valueText === null
         from: root.from
         to: root.to
-        onValueChanged: root.value = value
         stepSize: sliderControl.stepSize
         editable: true
     }
+
+    Binding on value {
+        when: sliderControl.pressed
+        value: sliderControl.value
+    }
+
+    Binding on value {
+        when: spinBox.up.pressed
+        value: spinBox.value
+    }
+
+    Binding on value {
+        when: spinBox.down.pressed
+        value: spinBox.value
+    }
+
+    Binding on value {
+        when: spinBox.focus
+        value: spinBox.value
+    }
```

Comment:
I believe that all spinbox binds could be replaced by `valueModified` signal

Reply:
I tried. does not work. the problem is that:

onValueModified: root.value = value <--- this breaks the binding

creating a Binding on value {} doesn't breaks dynamic bindings.

Reply:
I was talking about:
```
Binding on value {
	when: spinBox.valueChanged
    ...
}
```